src/main/java/roomescape/dto/ReservationRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package roomescape.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+@Getter
+@Setter
+public class ReservationRequestDto {
```

Comment:
```suggestion
public class ReservationRequest {
```

DTO 클래스의 이름 뒤에 Dto 가 추가적으로 붙는 것은 의미 중복일 수 있겠다는 생각이 들어요.
`~Request` or `~Response` 라는 이름으로도 요청/응답을 위한 DTO 객체구나! 라는 의미가 내포되어 있는 것 같다는 생각..인데요. 예린님은 이에 대해 어떻게 생각하시나요?

Reply:
항상 Dto를 작성하는 편이긴 했는데 Request, Response로 구분하는 것만으로 이해가 되는 거 같아서 그렇게 해도 좋을 거 같네요!

Reply:
정답이 없는 영역이긴 해서, 예린님이 잘 생각해보시고 무엇이 괜찮을지 선택하시면 될 것 같아요 !